Changes.md

```diff
@@ -7,6 +7,11 @@ Improvements
 - FileMenu : File loading and saving no longer locks the UI, and can be cancelled.
 - NodeEditor : The "Node Name" label is now draggable. For instance, it can be dragged to the PythonEditor to get a reference to the node or to the GraphEditor to find the node in the graph.
 
```

Comment:
Not sure where to leave this comment, but there's a typo in the commit message for "Widget : Fix drag handling bug":  I assume it should read "We now *abort*" rather than "We now about"

Reply:
Rebased and fixed in 1554982.